mmv1/templates/terraform/examples/bigquery_dataset_authorized_routine.tf.erb

```diff
@@ -0,0 +1,50 @@
+resource "google_bigquery_dataset" "public" {
+  dataset_id  = "<%= ctx[:vars]['public'] %>"
+  description = "This dataset is public"
+}
+
+resource "google_bigquery_routine" "public" {
+  dataset_id      = google_bigquery_dataset.public.dataset_id
+  routine_id      = "sample_routine"
+  routine_type    = "TABLE_VALUED_FUNCTION"
+  language        = "SQL"
+  definition_body = <<-EOS
+    SELECT 1 + value AS value
+  EOS
+  arguments {
+    name          = "value"
+    argument_kind = "FIXED_TYPE"
+    data_type     = jsonencode({ "typeKind" = "INT64" })
+  }
+  return_table_type = jsonencode({ "columns" = [
+    { "name" = "value", "type" = { "typeKind" = "INT64" } },
+  ] })
+}
+
+resource "google_bigquery_dataset" "private" {
+  dataset_id  = "<%= ctx[:vars]['private'] %>"
+  description = "This dataset is private"
+  access {
+    role         = "WRITER"
+    specialGroup = "projectWriters"
```

Comment:
this would need to be `special_group` to match the Terraform field name - but I'd prefer not including these `special_group` blocks. If we just match what the API is returning we won't be able to tell whether it's acting on what we send it or it's ignoring what we sent but it just happens to match what's in the config right now.

Reply:
@melinath 
Thank you for your comment.
Sorry for my misunderstanding about your advice.
I agree with your opinion.
I removed them.